icons/hadoop/hadoop-line.svg

```diff
@@ -0,0 +1,11 @@
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(0%,0%,0%);fill-opacity:1;" d="M 55.398438 34.101562 C 53.199219 34.699219 51.800781 36.101562 50.5 39 C 49.5 41.199219 49 43.300781 49.300781 43.699219 C 49.699219 44 50 43.601562 50 42.800781 C 50 41.898438 50.898438 40.199219 52 38.898438 C 53.800781 36.601562 53.898438 36.601562 54 38.5 C 54.101562 40.300781 54.199219 40.199219 55.5 38 C 56.199219 36.601562 58.101562 35 59.699219 34.300781 C 61.800781 33.398438 62 33.101562 60.5 33.101562 C 59.398438 33.199219 57.101562 33.601562 55.398438 34.101562 Z M 55.398438 34.101562 "/>
```

Comment:
plain and line icons should have only one path if possible. So please check if the paths can be merged into one :)